src/EnergyPlus/AirflowNetworkSolver.cc

```diff
@@ -3289,6 +3299,310 @@ Label90: ;
 	}
 
 	void
+	AFEOAF(
```

Comment:
I am guessing these are the function names from Airnet, but here they should have more descriptive names.  The original names can be referenced in the comments.